test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+git+https://github.com/OCA/bank-payment@refs/pull/873/head#subdirectory=setup/account_payment_partner
+git+https://github.com/OCA/bank-payment@refs/pull/870/head#subdirectory=setup/account_payment_mode
```

Comment:
```suggestion
odoo-addon-account_payment_partner @ git+https://github.com/OCA/bank-payment@refs/pull/873/head#subdirectory=setup/account_payment_partner
odoo-addon-account_payment_mode @ git+https://github.com/OCA/bank-payment@refs/pull/870/head#subdirectory=setup/account_payment_mode
```

Reply:
This can be removed as both PRs are now merged.